.dockerignore

```diff
@@ -1,5 +1,9 @@
-#ignore all files
-*/*
+#ignore all
+**
 
-#except release
-!/.build/release/*
```

Comment:
Here not sure if CI uses this .dockerignore file at all. Else we need to add back this line.

Reply:
Yes it does. Issue fixed by using 2 a Dockerfile-CI.dockergnore file.